book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,121 +1,389 @@
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+
+
+
+OSH projects emerge as **a response to a concrete need**,  which is reflected in the project lifecycle.
+For example, you may need to process lots of samples, or to measure a parameter you were not considering before, or to take measurements outside of the lab. 
+With the analyzing of your need, your OSH project starts.
+It is advisable to scope the project at this point, maybe looking for similar project one can join, instead of starting a new one.
+It may indeed be useful to draw a roadmap, look and find future users, contributors and manufacturers (who may have complementary skills to yours), and decide for a license.
+It is reasonable to invest quite a lot of time in looking for similar projects.
+OSH are often difficult to find and comprehend, but finding projects may save you a lot of time and hassle.
+Indeed, you may learn a lot about your needs, refine your ideas, and may even find collaborators while browsing existing OSH projects.
+
+
+
+You may then test some ideas to address your particular need, which is a process called **prototyping**.
+After many iterations and testing, you will be able to select a prototype for further development.
+The design that solves your need but is not yet complete nor stable is called a **demonstrator**.  
```

Comment:
```suggestion
The design that solves your need but is not yet complete nor ready to be replicated (usually because some parts are not well documented or requires a lot of manual adjustment) is called a **demonstrator**.  
```